setup.py

```diff
@@ -64,4 +64,9 @@
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
     ),
+    project_urls={
+        'Source Code': 'https://github.com/certifi/python-certifi',
+        'Documentation': 'https://certifiio.readthedocs.io/en/latest/',
+        'Bug Tracker': 'https://github.com/certifi/python-certifi/issues',
```

Comment:
"Issue [tT]racker" is about four times as common as "Bug Tracker": https://github.com/hugovk/pypi-tools/issues/11#issuecomment-549175199

```suggestion
        'Issue Tracker': 'https://github.com/certifi/python-certifi/issues',
```